extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -452,8 +452,8 @@
     "event": "https://tags-dmp.vpadn.com/et?t=_timestamp_&sdkn=j&sdkv=1.2.0&lk=!licence_key&en=UTF-8&ctid=_client_id_&ev=!ev_name&pl=!payload&is_amp=1&page_id=_page_view_id_"
   },
   "webengage": {
-    "base": "https://c.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&isAmp=1",
-    "wePageview": "https://c.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&eventName=Page Viewed&isAmp=1"
+    "base": "https://c.in.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&isAmp=1",
```

Comment:
`${region}` won't be resolved to `in` because there's no `vars` declared in the test. Instead, it will be resolved to `!region` in the test setting.

Reply:
@zhouyx Thanks, I think it worked. But some other error was thrown in the test, related to cross-origin. Can you please take a look? I don't think it is related to any of my changes!

Reply:
I've restarted the test and it passed.